docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,201 @@
+    print(key.name)
+    print(key.measure)
+
+::
+
+    population
+
+    structure
+
+Because we're looking at the 'population.structure' key, we only have a type and measure.
+
+
+Reading data
+-------------
+Now that we've seen how to create an :class:`~vivarium_public_health.dataset_manager.artifact.Artifact` object and
+view the underlying storage structure, let's cover how to actually retrieve data from that artifact. We'll use the
+:func:`~vivarium_public_health.dataset_manager.artifact.Artifact.load` method. We saw the key names in our artifact
+in the previous step, and we'll use those names to load data. For example, if we want to load the population structure
+data from our Artifact we do:
+
+.. code-block:: python
+
+    art = Artifact('test_artifact.hdf')
+    pop = art.load('population.structure')
+    print(pop.head()))
+
+::
+
+                                                                           value
+    age_group_end age_group_start location sex    year_end year_start
+    0.019178      0.0             Ethiopia Female 2007     2006        25610.50
+                                           Male   2012     2011        29136.66
+                                                  2009     2008        27492.91
+                                           Female 2000     1999        22157.50
+                                                  1993     1992        19066.45
+
+Notice that if we construct our artifact with filter terms as discussed above, we'll filter the data
+that gets loaded out of it:
+
+.. code-block:: python
+
+    art = Artifact('test_artifact.hdf', filter_terms=['age_group_start > 5'])
+    pop = art.load('population.structure')
+    print(pop.head()))
+
+::
+
+                                                                            value
+    age_group_end age_group_start location sex    year_end year_start
+    15.0          10.0            Ethiopia Male   2011     2010        6009393.00
+                                                  2003     2002        4489336.99
+                                           Female 2016     2015        6424674.99
+                                           Male   2017     2016        6610845.00
+                                           Female 2006     2005        4922733.99
+
+Writing data
+------------
+To write new data to an artifact, use the :func:`~vivarium_public_health.dataset_manager.artifact.Artifact.write` method,
+passing the full key (in the string representation we saw above of type.name.measure or type.measure) and the data you wish
+to store.
+
+.. code-block:: python
+
+    new_data = ['United States', 'Washington', 'California']
+
+    art.write('locations.names', new_data)
+
+    if 'locations.names' in art:
+        print('Successfully Added!')
+
+::
+
+    Successfully Added!
+
+If the key you wish to write to is already in the artifact, you'll want to use the :func:`~vivarium_public_health.dataset_manager.artifact.Artifact.replace`
```

Comment:
Definitely start the replace section with an error trying to write to an existing key.